sceptre/stack.py

```diff
@@ -421,7 +421,7 @@ def _set_field_with_deprecated_alias(
 
         if preferred_value and deprecated_value:
             raise InvalidConfigFileError(
-                f"Both '{preferred_config_name}' and '{deprecated_config_name}' are set; You should only set a "
+                f"Both '{preferred_config_name}' and '{deprecated_config_name}' are set, You should only set a "
```

Comment:
Is it related to the Docker update, or opportunistic? (If it's opportunistic, I think a period (`.`) works best here.)